File: 59A.c
#include "stdio.h"

int _length(char arr[]) {
    int count = 0;
    while(arr[count]) count++;
    return count;
}

int main() {
    char word[110];
    int count = 1;
    scanf("%s",word);
    int i = 0;
    while(word[i]) {
        if((int)word[i] >= 97) count++;
    }

    if(count > _length(word)-count) {printf();}
}

File: 59A.py
word = input()

def caps(word:str):
    count = 0
    for i in list(word):
        if (ord(i) >= 97):
            count += 1
    
    if count >= len(word)-count:
        return False
    else: return True

if caps(word): print(word.upper())
else: print(word.lower())